thorlcr/activities/funnel/thfunnelslave.cpp

```diff
@@ -682,6 +680,8 @@ class NonEmptySlaveActivity : public CSlaveActivity
         sendReceiving = false;
         masterMpTag = TAG_NULL;
         appendOutputLinked(this);
+        if (container.queryLocalOrGrouped())
```

Comment:
The setRequireInitData(false) call will prevent the recvActivityInitData() requesting initialization data from the master.

It only needs it in the global case, that's what the init(MemoryBuffer &data, MemoryBuffer &slaveData) below this does - deserialize a mpTag tag used by global only.